Add `GB Cup 2023 Autumn` wiki articles (#10075)
ppy/osu-wiki: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md

### Stage instructions

1. Following the Qualifier stage, all players will participate in a phase to form teams, called the *drafting phase*. Both divisions will have independent drafting phases. All unpicked players will be eliminated from the tournament.
2. Following the drafting phase, a Swiss stage will be played in both zones, called the *regular stage*. 16 teams will advance in total, and the others get eliminated from the tournament.
3. Following the regular stage, the elimination stage will be played, called *playoffs*. The top 8 teams from the regular stage will advance to a winner bracket, other teams will get into loser bracket.
4. All losing teams in the loser bracket will be eliminated from the tournament and the last team will be the winner.

### Drafting phase instructions

1. The host will declare one captain for each team 24 hours before the drafting phase.
   - Drafting order will also be declared by the host.
   - Captains are selected by performance and preference.
   - Non-captains will join a group of players waiting to join a team, herein called a "player pool".
2. Captains decide their team members during the drafting phase.

>    - Captains are allowed to conduct and coordinate "tryouts", tests to gauge player aptitude, on their own terms. 

Reviewer: Untranslated, but dosen't matter i guess. this is just an reminder for captains.

Reviewer: fixed in 689aebb

3. There will be a 120-second planning phase before the picking phase starts.
4. During the picking phase, the pick order consists of 2 phases that repeat twice, such that every captain gets to choose 4 players.
   - During phase 1 and 3, picks start in *increasing* order, from the 1st to the 24th captain.
   - During phase 2 and 4, picks start in *decreasing* order, from the 24th to the 1st captain.
   - *This might change depending on the number of incoming registrations.*
5. Each captain has 60 seconds to pick one player from the player pool on their turn. 
   - If the captain does not pick a player within 60 seconds, a random player from the remaining player pool will be assigned to their team.

> 6. If the captain picks a player, the pick time still has to elapse before the next captain's turn starts, meaning each phase will take 16 minutes.

Reviewer: Untranslated.

Reviewer: fixed in 689aebb

   - The room settings are `Game mode: "osu!mania"`, `Team mode: "Team Vs"` and `Score mode: "ScoreV2"`. The room name must follow the pattern of `GBC 2023 Autumn: ({TeamRed}) vs ({TeamBlue})`.
   - The team mentioned first in the room name must be the red team, and the team mentioned second in the room name must be the blue team.
2. Each captain can protect **one beatmap** from the pool. These beatmaps cannot be banned. Then, each captain can ban **one beatmap** from the pool. These beatmaps cannot be picked by any team for the entire duration of the match.
3. Both captains will alternate picking a beatmap from the mappool.
4. Each captain must use `!roll` once in `#multiplayer`.
   - The winner of the `!roll` may choose to pick first or ban first in the match. The other choice will be used by loser of the roll. The team who picks first will also protect first, and the others will ban first.
5. After each pick, both captains **may** ban **one** of the opposing team members from playing the beatmap in a match, **except during tiebreakers**.
   - Any player can only be banned like this **once** per match.

>    - Only players who have already played in the match can be banned.

Reviewer: Untranslated, is this rules changed?

Reviewer: fixed on [3704f72](https://github.com/ppy/osu-wiki/pull/10075/commits/3704f7262ef29ea324bb6e985a23fd11d3930204)

   - Picking beatmaps and banning players should be sent **using a single message**. If a team continues to be slow at banning, referees have rights to **ignore the ban**.
   - For the first occurrence:
     - The team will receive a verbal warning from the referee.
   - On subsequent occurrences:
     - For bans of opposing players, the captain forfeits the ban.
     - For map bans, the captain forfeits the ban.
     - For map picks, a random map will be chosen from the mappool using `!roll X`, where X is the number of maps that were neither picked nor banned, excluding the tiebreaker.
     - For a ready timer, the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
   - Repeat offenders may receive further sanctions from the host.
8. Each team will receive **one** "tactical timeout" of **two minutes**, to be used as extra time to pick or ban a beatmap. *The tactical timeout is optional.*

>    - The tactical timeout may be called on a tiebreaker, provided that both teams' captains agree to it.

Reviewer 2: this rule conflicts with rule 6. please remove one of them and adjust accordingly

### Mappool instructions

1. There will be five separate mappools. *This might change depending on the number of incoming registrations.*

>    - Qualifiers will use a separate mappool.
>    - Regular stage Round 1 and Round 2 use mappool \#1.
>    - Regular stage Round 3 and Round 4 use mappool \#2.
>    - Round of 16 and Quarterfinals use mappool \#3.
>    - Semifinals and Finals use mappool \#4.
>    - This might change.

Reviewer 2: ```suggestion
   - Qualifiers will use a separate mappool.
   - Regular stage round 1 and round 2 use mappool #1.
   - Regular stage round 3 and round 4 use mappool #2.
   - Round of 16 and Quarterfinals use mappool #3.
   - Semifinals and Finals use mappool #4.
```

2. Each mappool consists of a fixed amount of maps each stage which will all be played under [FreeMod](/wiki/Gameplay/Game_modifier#freemod) conditions. This means that there is a unique FreeMod bracket.
3. The mappool sizes are as follows:
   - Qualifiers: 8 beatmaps   

>    - Mappool \#1 & \#2: 15 beatmaps
>    - Mappool \#3 & \#4: 17 beatmaps.

Reviewer 2: ```suggestion
   - Mappool #1 & #2: 15 beatmaps
   - Mappool #3 & #4: 17 beatmaps.
```

4. Each mappool has one tiebreaker, except for the Qualifiers.
5. Possible mod choices for the FreeMod bracket are `NoFail`, `Hidden`, `Fade In`, `Flashlight`, and `Mirror`.
6. The tiebreaker will be played under FreeMod conditions.

### Scheduling information

1. Each stage will be held on **a single weekend**.
2. Matches in Qualifiers will be held during specific time slots proposed by the tournament staff. Each player is free to choose a time slot in which they want to play.

>    - **Players that don't pick a time slot by Wednesday, 11 October 23:59 UTC+8 will be disqualified from the competition.**

Reviewer 2: ```suggestion
   - **Players that do not pick a time slot by Wednesday, 11 October 23:59 UTC+8 will be disqualified from the competition.**
```

3. All matches will be held between Friday 18:00 and Sunday 23:59 (UTC+8).

>    - Rescheduling a match to a weekday is only permitted if **both** teams request and agree to it.

Reviewer 2: ```suggestion
   - Rescheduling a match to a weekday is only permitted if absolutely necessary, **both** teams request and agree to it, and staff members are available.
```

4. Scheduling will be handled by the host. Schedules will be released on the Sunday before the first match of the stage. The host will try to create a schedule that respects all participants' preferences.
   - Please inform the host before Sunday if you expect a specific time slot to be unavailable in the following week. The host will try to accommodate all wishes, but makes no promises.
5. **Reschedules will only be considered if both teams agree to a time and communicate it to the tournament staff, before Wednesday 23:59 UTC+8 of the week the match is to take place on.**
   - Reschedules may only be requested by a team captain.

>    - Matches may not be scheduled to any time beyond **Sunday, 23:59 UTC+8** of the week they are to be played at.
>      - ... Unless its absolutely necessary, **both** teams request and agree to it, and staff avaliable.

Reviewer 2: already mentioned in rule 3 above

```suggestion
```

5. There will be an optional 5-minute break in between the first and second play-through of the mappool.
6. We suggest players not to broadcast or share their results to avoid seed manipulation.
7. Qualifier stage's result will using to seeding and choosen captains.
   - Higher scores lead to a higher chance to be picked during drafting phase. Players are eliminated if they do not get picked, or if they get too low of a score during Qualifers.
8. Some best-performance players will advance to drafting phase.
   - Disrupting the match by foul play or seed manipulation will result in disqualification, and any such players will be announced before the drafting phase takes place.
9. Players will use same multi rooms if they select same time to play qualifier. It is prohibited to abort a match with the reason of technical problem. With this situation, the score will be recorded normally.

### Stage instructions

1. Following the Qualifier stage, all player should participate a phase to form teams , called *Drafting phase*. Both zones will have their independent drafting. All the unpicked player will be eliminated from the tournament.

> 2. Following the drafting phase, a swiss stage will be played in both zones, called the *Regular stage*. There will be `16` teams advance in total, and the others get eliminated from the tournament.

Reviewer 2: ```suggestion
2. Following the drafting phase, a Swiss stage will be played in both zones, called the *regular stage*. 16 teams will advance in total, and the others get eliminated from the tournament.
```

> 3. Following the Regular stage, the Elimination stage will be played, called *Playoffs*. Top 8 teams in regular stage will get into winner bracket, other teams will get into loser bracket.

Reviewer 2: ```suggestion
3. Following the regular stage, the elimination stage will be played, called *playoffs*. The top 8 teams from the regular stage will advance to a winner bracket, other teams will get into loser bracket.
```

> 4. All the losing team in loser bracket will be eliminated from the tournament, the last team will be the winner.

Reviewer 2: ```suggestion
4. All losing teams in the loser bracket will be eliminated from the tournament and the last team will be the winner.
```
